examples/source_stream_wfs_3d.html

```diff
@@ -197,7 +202,6 @@
             view.addEventListener(itowns.GLOBE_VIEW_EVENTS.GLOBE_INITIALIZED, function () {
                 // eslint-disable-next-line no-console
                 console.info('Globe initialized');
-                view.controls.setTilt(45);
             });
 
             debug.createTileDebugUI(menuGlobe.gui, view);
```

Comment:
You need to do something for the `view.controls.isPaused()` below.

Reply:
done